docs/source/models/atomic_physics.rst

```diff
@@ -71,9 +71,26 @@ Changes of the PIC-simulation state by the atomic physics/FLYonPIC step are prop
 
 Which transitions and states are included in this update is user configurable, with the user providing all transition coefficients and state energies via an input file.
 
+Getting Started With FLYonPIC
+-----------------------------
+
+To use FLYonPIC in a simulation setup, you need to provide a set of atomic input data files, see :ref:`atomicPhysicsInputData` and mark at least one species as an atomic physics ion species and one species as an atomic physics electron species.
+
+See the the compile time tests of atomic physics, `debug compile test <https://github.com/ComputationalRadiationPhysics/picongpu/tree/dev/share/picongpu/tests/compileSparser>`_ and `compile sparser example <https://github.com/ComputationalRadiationPhysics/picongpu/tree/dev/share/picongpu/tests/compileAtomicPhysics>`_ for an example using atomic physics.
```

Comment:
Why's "compile time tests of atomic physics" not a link?

Reply:
good suggestion, fixed